SPRINTUSER_README.md

```diff
@@ -57,10 +59,10 @@ Extract the drupal_sprint_package.RELEASE.zip file, and open or browse to the "d
 
 **⚠️Docker for Windows**: You *must* share the C: drive (preferably all drives, or any other drive your home directory may be on) under Docker->Settings->Shared Drives.
 
-**⚠️All users:** additional information is available at [ddev docker instructions](https://ddev.readthedocs.io/en/latest/users/docker_installation/).
-
 **⚠️Linux users:** You'll probably need the [ddev docker instructions](https://ddev.readthedocs.io/en/latest/users/docker_installation/) to get docker properly set up.
 
+**⚠️All users:** Now start the Docker app you installed. If Docker asks you to login, you can ignore this.
```

Comment:
This actually is misleading. On WIndows and MacOS docker always asks you to log into the *machine* in order to have privs on the filesystem, etc.

Reply:
I think the intent here is that Docker for * now is persistent in asking the user to create a Docker Hub user account (which isn't necessary). Some users get stuck on this because they think they have to create one and login to use Docker.

Reply:
I do think that's the intent (although I haven't noticed it being obnoxious). But it's important to make the differentiation between logging into the machine (for docker install) and logging into dockerhub.